R/join.R

```diff
@@ -4,26 +4,30 @@ insensitive_join <- function(fun = dplyr::left_join, type = "LEFT") {
   new_fun <- fun
   body(new_fun) <- substitute({
     by <- dplyr:::common_by(by, x, y)
-    
+
     tmp_by_x <- paste0("_", by$x, "_")
     tmp_by_y <- paste0("_", by$y, "_")
     for (i in seq_along(by$x)) {
       x[[tmp_by_x[[i]]]] <- tolower(x[[by$x[[i]]]])
       y[[tmp_by_y[[i]]]] <- tolower(y[[by$y[[i]]]])
       y[[by$y[[i]]]] <- NULL
     }
-    
-    res <- fun(x, y, list(x = tmp_by_x, y = tmp_by_y))
+
+    res <- fun(x, y, list(x = tmp_by_x, y = tmp_by_y), suffix = suffix)
```

Comment:
suffix seems to be undefined in this function.

Reply:
Also, shouldn't the copy argument passed down too?

Reply:
Add comment for for suffix and copy arguments.